src/modules/olMap/services/Mfp3Encoder.js

```diff
@@ -282,17 +293,16 @@ export class BvvMfp3Encoder {
 			case GeoResourceTypes.WMS:
 				return this._encodeWMS(encodableLayer, geoResource, groupOpacity);
 			case GeoResourceTypes.VT:
-				console.warn(`VectorTiles are currently not supported by MFP. Missing substitution for GeoResource '${geoResource.id}'.`);
-				return [];
+				return await this._encodeVectorTiles(encodableLayer, groupOpacity);
 			default:
 				return false;
 		}
 	}
 
-	_encodeGroup(groupLayer, encodingErrorCallback) {
+	async _encodeGroup(groupLayer, encodingErrorCallback) {
 		const subLayers = groupLayer.getLayers().getArray();
 		const groupOpacity = groupLayer.getOpacity();
-		return subLayers.map((l) => this._encode(l, encodingErrorCallback, groupOpacity));
+		return await Promise.all(subLayers.map(async (l) => await this._encode(l, encodingErrorCallback, groupOpacity)));
```

Comment:
`return await Promise.all(subLayers.map(async (l) => this._encode(l, encodingErrorCallback, groupOpacity)));`